applications/asset_tracker/src/main.c

```diff
@@ -1631,6 +1672,13 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 		if (err) {
 			LOG_ERR("Error processing PGPS packet: %d", err);
 		}
+#endif
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
+		/* log the location response from the cloud,
+		 * if configured in the request
+		 */
+		LOG_INF("rcvd data: %s, len: %d", log_strdup(evt->data.msg.buf),
```

Comment:
you can use `nrf_cloud_parse_cell_pos_response` here and if it returns 1, it was not cell_pos data, so continue to process.